backend/src/akvo/lumen/endpoint/invite.clj

```diff
@@ -1,28 +1,40 @@
 (ns akvo.lumen.endpoint.invite
   (:require [akvo.lumen.component.tenant-manager :refer [connection]]
-            [akvo.lumen.component.user-manager :as user-manager]
             [akvo.lumen.http :as http]
+            [akvo.lumen.lib.user :as user]
             [compojure.core :refer :all]))
 
 
-(defn endpoint [{:keys [tenant-manager user-manager]}]
-  (context "/api/admin/invites" {:keys [jwt-claims params server-name tenant]}
+(defn redirect-location
+  "Unfortually local development is done by running the api at port 3000
```

Comment:
But from the users point of view they interact with our web app via one port 433. In our dev setup you need to jump between 3000& 3030. Makes sense?